src/adagrams.js

```diff
@@ -1,15 +1,137 @@
+// import { all } from "core-js/fn/promise";
+
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+//Helper Function................
+const createWordList = function (word) {
+  let wordlist = [];
+  for (let each of word) {
+    wordlist.push(each.toUpperCase());
+  }
+  return wordlist;
+};
+
+//...........................
+
+// ..........wave1...............
 export const drawLetters = () => {
```

Comment:
⚠️ This has a subtle issue.  Drawing a "Z" is equally possible to drawing an "A" despite there being 9 As in the pool and only 1 Z.